org.freedesktop.Sdk.json.in

```diff
@@ -38,7 +38,9 @@
             "subdirectories": true,
             "subdirectory-suffix": "gtk-3.0",
             "no-autodownload": true,
-            "version": "3.22"
+            "version": "3.22",
+            "download-if": "active-gtk-theme",
+            "enable-if": "active-gtk-theme"
```

Comment:
I don't think enable-if should be set. There is no gain to not having the theme available if installed, even if its not the current theme. And it can be useful, for instance if the auto-picking of themes is wrong or if you want to change theme.